packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/android/signout.md

```diff
@@ -2,7 +2,7 @@
 
 The sign-out flow for a Classic Engine Java Auth SDK app typically involves deleting any persistent session storage and redirecting the user to a sign-out page.
 
-### Okta Identity Engine SDK authentication flow for basic sign-out
+### Identity Engine SDK authentication flow for basic sign-out
```

Comment:
Both headings:
Okta Authentication SDK authentication flow for basic sign-out
and
Identity Engine SDK authentication flow for basic sign-out

the style is to not use the term sign-in and sign-out as a noun. It's used as an adjective.

I struggled to figure out a better title that didn't repeat both authentication and flow

Reply:
In the next commit, I'll be changing them to 

Okta Authentication SDK basic sign-out flow

and

Identity Engine SDK basic sign-out flow